projects/packages/videopress/src/client/admin/components/admin-page/libraries.tsx

```diff
@@ -61,7 +55,7 @@ const VideoLibraryWrapper = ( {
 				<Text>{ totalVideos } Video</Text>
 				{ hideFilter ? null : (
 					<div className={ styles[ 'filter-wrapper' ] }>
-						<SearchInput onChange={ handleSearchChange } onEnter={ handleSearch } />
+						<SearchInput onSearch={ setSearch } />
```

Comment:
I suggest to add `onEnter` as well, because I would expect Enter to trigger the search.

```suggestion
						<SearchInput onSearch={ setSearch } onEnter={ setSearch }/>
```

Reply:
Hmm, forget it. That way the search would happen twice.

Reply:
Or maybe `onEnterHandler` should not call `debouncedOnChange`?

Reply:
the Component handles that internally, meaning that it will trigger onSearch() when onEnter() and onChange() happens.

Reply:
yes, typing and pressing ends up with a double request